python/lancedb/remote/__init__.py

```diff
@@ -0,0 +1,66 @@
+#  Copyright 2023 LanceDB Developers
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+try:
+    import aiohttp
+    import pydantic
+except ImportError:
+    raise ImportError(
+        "LanceDB remote client requires aiohttp and pydantic, please install lancedb[managed]"
+    )
+
+import abc
+from typing import List, Optional
```

Comment:
shall we run `isort` over this file?

Reply:
yes, after this PR. will submit an isort PR. there are some left over stuff from previous PR.